packages/renderer/src/App.spec.ts

```diff
@@ -76,3 +85,28 @@ test('test /images/:id/:engineId route', async () => {
   await tick();
   expect(mocks.ImagesList).toHaveBeenCalled();
 });
+
+test('recieve context menu visible event from main', async () => {
+  render(App);
+  // send 'context-menu:visible' event
+  messages.get('context-menu:visible')?.(true);
+
+  await new Promise(resolve => setTimeout(resolve, 100));
+
+  //  check if send method has been called
+  expect(dispatchEventMock).toHaveBeenCalledWith(new Event(''));
```

Comment:
you can use expect.any() if you don't want to check with a new object or expect.anything or expecT.objectContaining etc to test some properties in the object

no need to create a plain Event object